I am receiving "Error getting Block Prices From Polygon" message.

### Discord Discussion Link

https://discord.com/channels/808358975287722045/888500174685614090/1118973511113912331

### What browsers are you seeing the problem on?

Chrome

### What were you trying to do?

I have tried a few small swaps on Polygon network (USDC for POOL and USDC for Matic). I launched the extension, selected Polygon, and attempted a swap. The estimated swap fee was only a few cents, and I made sure I had a few dollars (matic) in the wallet. 

### What did not work?

Each attempt has failed to broadcast. I have received an "Error broadcasting" and "Error getting block prices from Polygon"

Also to note: the imbedded website link "https://gasstation-mainnet.matic.network/v2" does not seem t work when I loaded it into.  

### Version

v0.36.0

### Relevant log output

```shell
Prior to approving asset: 
logger.ts:208 Error getting block prices from Polygon Error: missing response (requestBody=null, requestMethod="GET", serverError={}, url="https://gasstation-mainnet.matic.network/v2", code=SERVER_ERROR, version=web/5.5.1)
    at Logger.makeError (index.js:185:23)
    at Logger.throwError (index.js:194:20)
    at index.js:205:32
    at Generator.throw (<anonymous>)
    at rejected (index.js:6:57)
logEvent @ logger.ts:208
error @ logger.ts:170
getBlockPrices @ gas.ts:185
await in getBlockPrices (async)
pollBlockPricesForNetwork @ index.ts:1297
(anonymous) @ index.ts:1276
pollBlockPrices @ index.ts:1275
handler @ index.ts:299
handleAlarm @ base.ts:155

After broadcasting: 
Error broadcasting transaction {hash: '0x2f4299ef5e5dedb24fff901c2b16009095586bf2e0dd732453004f518501d4a7', from: '0xBe2063F9335F3A8989fCa057Eaa96455BFa027aE', to: '0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174', nonce: 95, input: '0x095ea7b3000000000000000000000000def1c0ded9bec7f1…fffffffffffffffffffffffffffffffffffffffffffffffff', …} Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":88,\"error\":{\"code\":-32000,\"message\":\"transaction underpriced\"}}", error={"code":-32000}, requestBody="{\"method\":\"eth_sendRawTransaction\",\"params\":[\"0x02f8b281895f844190ab00853c9428b40083010100942791bca1f2de4661ed88a30c99a7a9449aa8417480b844095ea7b3000000000000000000000000def1c0ded9bec7f1a1670819833240f027b25effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffc001a07e83213ed1f8f1d6dea3e34da3c51d73dc461ad54075e32834a9dde4bd61695da00a689d4feb7464ee0db27751a93cd11792665ee3a28c559114e43d7549218e40\"],\"id\":88,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://polygon-mainnet.g.alchemy.com/v2/M9RcHmESdhvr_WIj3cfiau3iI98yvY2k", code=SERVER_ERROR, version=web/5.5.0)
    at Logger.makeError (index.js:185:23)
    at Logger.throwError (index.js:194:20)
    at index.js:213:32
    at Generator.next (<anonymous>)
    at fulfilled (index.js:5:58)
```

--- a/background/lib/gas.ts
+++ b/background/lib/gas.ts
@@ -42,7 +42,7 @@ type PolygonGasResponse = {
 const getPolygonGasPrices = async (price: bigint): Promise<BlockPrices> => {
   // @TODO Validate this response using ajv
   const gasEstimates = (await fetchJson(
-    "https://gasstation-mainnet.matic.network/v2"
+    "https://gasstation.polygon.technology/v2"
   )) as PolygonGasResponse
 
   const baseFeePerGas = BigInt(Math.ceil(gasEstimates.estimatedBaseFee * 1e9))

--- a/background/services/indexing/index.ts
+++ b/background/services/indexing/index.ts
@@ -48,7 +48,6 @@ import {
   normalizeEVMAddress,
   sameEVMAddress,
 } from "../../lib/utils"
-import { fixPolygonWETHIssue, polygonTokenListURL } from "./token-list-edit"
 
 // Transactions seen within this many blocks of the chain tip will schedule a
 // token refresh sooner than the standard rate.
@@ -911,11 +910,6 @@ export default class IndexingService extends BaseService<Events> {
           try {
             const newListRef = await fetchAndValidateTokenList(url)
 
-            if (url === polygonTokenListURL) {
-              newListRef.tokenList.tokens = fixPolygonWETHIssue(
-                newListRef.tokenList.tokens
-              )
-            }
             await this.db.saveTokenList(url, newListRef.tokenList)
           } catch (err) {
             logger.error(

--- a/background/services/indexing/token-list-edit.ts
+++ b/background/services/indexing/token-list-edit.ts
@@ -34,7 +34,7 @@ export function fixPolygonWETHIssue(tokensParam: TokenInfo[]): TokenInfo[] {
       ...tokens[tokenToRenameIndex],
       name: "Wrapped Ether",
       symbol: "WETH",
-      logoURI: "https://wallet-asset.matic.network/img/tokens/weth.svg",
+      logoURI: "https://assets.polygon.technology/tokenAssets/eth.svg",
     }
   }
 

